it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/trustControllers/errorControllers/CannotConfirmBusinessControllerISpec.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.errorControllers
+
+import play.api.libs.ws.WSResponse
+import play.api.test.Helpers.{NO_CONTENT, OK, SEE_OTHER, await, defaultAwaitTimeout}
+import uk.gov.hmrc.minorentityidentificationfrontend.assets.TestConstants.{testContinueUrl, testInternalId, testJourneyId, testTrustsJourneyConfig}
+import uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.{routes => trustRoutes}
```

Comment:
As all the routes referred to below are in the same package you could use "import uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.routes" and the default "routes".

Reply:
It complained when I tried this and was still only using routes from the errorController package even though I'd specified the import above so I've left it as it was

Reply:
Ok